### DISCUSSION TITLE
Update the MicroserviceDemo only in the commercial version ?

### DISCUSSION THREAD

**Discussion by cld0815** [abpframework/abp#7535] [General] [answered]
The MicroserviceDemo has not been updated since version 3.2.1
but Commercial 4.2 has a Microservice Startup Template :
https://docs.abp.io/en/commercial/4.2/startup-templates/microservice/index

**Comment by maliming** [comment, marked as answer]
It is in the plan.

**Reply by cld0815** [reply]
OK, Many thanks

**Reply by cld0815** [reply]
> It is in the plan.

Could you please tell us when it will be updated

**Reply by gterdem** [reply]
It is updated with 4.3rc1 and will be updated like other startup templates.
